torch/onnx/_internal/exporter/_registration.py

```diff
@@ -64,11 +64,11 @@ def __post_init__(self) -> None:
                 if isinstance(self.onnx_function, onnxscript.OnnxFunction):
                     signature = _schemas.OpSignature.from_function(  # type: ignore[attr-defined]
                         self.onnx_function,
-                        # pyrefly: ignore  # missing-attribute
+                        # pyrefly: ignore [missing-attribute]
```

Comment:
Is the idea to replace mypy, or have both linters running?

Reply:
The idea is to eventually replace mypy, fixing these syntax issues first